### ISSUE TITLE
kitpvp prevents TNT damage on mobs

### ISSUE THREAD

**Issue by readme55** [cervinakuy/KitPvP#106]
tested on fresh vanilla spigot-paper 16.4 server with only essentials/vault plugin:

Without the kitpvp plugin TNT will behave like expected:
- destroy blocks
- hurt/kill mobs
- hurt myself the player

After adding the kitpvp plugin and executing `/kp addspawn` on my world:
- normal tnt wont hurt mobs anymore
- destroy blocks 
- hurt myself 

im trying to add mobs to kitpvp arena, but the "Bomber" class with TNT cant hurt the mobs.

I tested with the normal TNT and the kitpvp throwable TNT and both dont do damage to "zombie" after i set the kitpvp spawn. After removing the spawn with `/kp delarena`, TNT will work normal again right away!

**Comment by readme55** [comment]
thanks for advice on discord. I commented these lines: https://github.com/cervinakuy/KitPvP/blob/44a931135580fbc4cc89614a8e1bb3ec4d65217f/src/main/java/com/planetgallium/kitpvp/listener/ArenaListener.java#L135-L141

```
			} else if (e.getEntity() instanceof Damageable && e.getEntity().getType() != EntityType.PLAYER) {

				if (e.getDamager().getType() == EntityType.PRIMED_TNT) {

					e.setCancelled(true);

			       }
```
and now mobs get damaged from tnt!

**Comment by cervinakuy** [comment]
Thanks. I will still have to go in and add some code to ensure that other entities such as paintings and signs do not get destroyed with TNT (the original reason for that code).

I’ll probably end up doing something with LivingEntity but in the mean time I will keep this issue open as a reminder.

Thanks again for bringing this to my attention.

**Comment by cervinakuy** [comment]
Fixed in [7de7e17](https://github.com/cervinakuy/KitPvP/commit/7de7e1781db0688b29de76af7c095c9144871b63). This will be included in the next update.
